vendor/ansible_collections/nvidia/bare_metal/plugins/modules/instance_batch.py

```
@@ -246,6 +253,7 @@
 )),
 operating_system_id=dict(type='str'),
 phone_home_enabled=dict(type='bool'),
+secondary_vpc_ids=dict(type='list', elements='str'),
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)